invokeai/app/services/invocation_stats.py

```diff
@@ -115,6 +125,9 @@ class NodeStats:
     calls: int = 0
     time_used: float = 0.0  # seconds
     max_vram: float = 0.0  # GB
+    cache_hits: int = 0
+    cache_misses: int = 0
+    cache_high_watermark: int = 0
 
 
```

Comment:
I think this is missing `ram_used` and `ram_changed`?
```suggestion
    cache_hits: int = 0
    cache_misses: int = 0
    cache_high_watermark: int = 0
    ram_used: float = 0
    ram_changed: float = 0
```

Reply:
`ram_used` and `ram_cached` should not have been in `NodeStats` in the first place and have been removed.